site/profile/manifests/ccs/nvidia.pp

```diff
@@ -40,7 +40,7 @@
       default: {}
     }
 
-    if $facts['efi'] {
+    if fact('efi') {
```

Comment:
I would not change this. the `fact()` call is a function call which is slightly more expensive. even if `$facts['efi']` isn't set, it will return undef, i won't fail.

Reply:
I'm aware it adds the overhead of a function call.  I prefer to use the dot notation as the same format works for both hiera and puppet code.  It also reduces the number of chars in a line and I find more compact code is easier to read.